src/maintainer/pinning_deps.rst

```diff
@@ -3,6 +3,20 @@
 Pinned dependencies
 *******************
 
+**Pinning a dependency** is a practice of explicitly pointing to the version of a library which our package is linked with. 
+A pinned dependency , for example, can look like this : ``numpy==1.1.3`` . On the other hand, a non-pinned dependency would be  just: ``numpy``.
+
+**Why to pin your dependencies?**
+
+Explicitly declaring the versions of the dependencies could be advantageous to the quality of the software and to the developers and the open source community that makes up the software ecosystem.
+ 
+* Pinning our dependencies may help in avoiding a situation where our software does not builds or runs due to the release of newer versions of the dependencies which are incompatible with our software, consisting some breaking changes.
```

Comment:
```suggestion
* Pinning our dependencies may help in avoiding a situation where our software does not build or run due to the release of newer versions of the dependencies which are incompatible with our software, consisting some breaking changes.
```

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.